openfecwebapp/templates/partials/legal-search-results-advisory-opinion.html

```diff
@@ -1,8 +1,18 @@
-<tr>
-  <td><a title="{{ advisory_opinion.description }}" href="http://saos.fec.gov/aodocs/{{ advisory_opinion.no }}.pdf#search={{ query }}">{{ advisory_opinion.no }}</a></td>
+<tr class="legal-search-result">
+  <td><a title="{{ advisory_opinion.description }}" href="{{ advisory_opinion.pdf_url }}">AO {{ advisory_opinion.no }}</a></td>
   <td>{{ advisory_opinion.name }}</td>
   <td>
     <strong>{{ advisory_opinion.summary }}</strong>
-    <div class="t-serif" title="{{advisory_opinion.text}}">{{ advisory_opinion.text|truncate }}</div>
+    <div class="t-serif legal-search-result__hit">
+      &hellip;
+      {% for highlight in advisory_opinion.highlights %}
+        {{ highlight|safe }} &hellip;
+      {% endfor %}
+    </div>
+    {% if legal_include_display_all %}
+    <div class="legal-search-result__display-all">
+      <a href="/legal/advisory-opinions?search={{ query }}">Display all hits</a>
```

Comment:
Great point. I believe the intention here is that "Display all hits" doesn't actually take you to the advisory-opinion search, but it really just expands the row to show you all the hit text. I misunderstood the design and just included all the hit text concatenated with ellipsis. https://github.com/18F/fec-eregs/issues/76 will address this, but for now I think it's okay to go to the advisory opinion page. We can also remove this link if it's misleading.

cc @edmullen